pkg/action/actioncontroller.go

```diff
@@ -159,7 +159,8 @@ func (ac *ActionController) handle(ctx context.Context, action *storkv1.Action)
 			ac.activateClusterDuringFailback(action, true)
 		}
 	default:
-		ac.updateStatus(action, storkv1.ActionStatusFailed)
+		action.Status.Status = storkv1.ActionStatusFailed
+		ac.updateAction(action)
```

Comment:
Couldn't make sense of this change. The `updateStatus` is also updating the status of the `action` object right? Or am I missing something @diptiranjanpx?